src/main/java/edu/cornell/kfs/module/purap/businessobject/IWantNote.java

```diff
@@ -0,0 +1,34 @@
+package edu.cornell.kfs.module.purap.businessobject;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+public class IWantNote extends PersistableBusinessObjectBase {
```

Comment:
similar thought I had on the the SQL PR, should this be named something like IWantNoteDefault or something so it doesn't like this is the actual note text on the Iwant Document itself?

Reply:
Good point. How about IWantNoteDropdownValue ?